projects/flow.aggregation-mpi/project.py

```diff
@@ -0,0 +1,42 @@
+from flow import FlowProject, aggregator, directives
+
+
+class Project(FlowProject):
+    pass
+
+
+RANKS_PER_JOB = 2
+JOBS_PER_AGGREGATE = 8
+
+
+@Project.operation
+@directives(nranks=lambda *jobs: RANKS_PER_JOB * len(jobs))
+@aggregator.groupsof(JOBS_PER_AGGREGATE)
+def do_mpi_task(*jobs):
+    from mpi4py import MPI
+
+    comm = MPI.COMM_WORLD
+    rank = comm.Get_rank()
+    size = comm.Get_size()
+    num_jobs = len(jobs)
+    print(f"Launching MPI tasks for {len(jobs)} jobs. Rank {rank} of {size}.")
```

Comment:
```suggestion
    if world_rank == 0:
        print(f"Launching MPI tasks for {len(jobs)} jobs. Rank {rank} of {size}.")
```
Print this message only once.